src/app/(sidebar)/account/saved/page.tsx

```diff
@@ -0,0 +1,238 @@
+
+  const SavedKeypair = ({ keypair }: { keypair: SavedKeypair }) => {
+    return (
+      <Box gap="sm" addlClassName="PageBody__content">
+        <Input
+          id={`saved-kp-${keypair.timestamp}-name`}
+          fieldSize="md"
+          value={keypair.name}
+          readOnly
+          leftElement="Name"
+          rightElement={
+            <InputSideElement
+              variant="button"
+              placement="right"
+              onClick={() => {
+                setCurrentKeypairTimestamp(keypair.timestamp);
+              }}
+              icon={<Icon.Edit05 />}
+            />
+          }
+        />
+
+        <Input
+          id={`saved-kp-${keypair.timestamp}-pk`}
+          fieldSize="md"
+          value={keypair.publicKey}
+          readOnly
+          leftElement="Public"
+          copyButton={{ position: "right" }}
+        />
+
+        <Input
+          id={`saved-kp-${keypair.timestamp}-sk`}
+          fieldSize="md"
+          value={keypair.secretKey}
+          readOnly
+          leftElement="Secret"
+          copyButton={{ position: "right" }}
+          isPassword
+        />
+
+        <Box
+          gap="lg"
+          direction="row"
+          align="center"
+          justify="end"
+          addlClassName="Endpoints__urlBar__footer"
+        >
+          <SavedItemTimestampAndDelete
+            timestamp={keypair.timestamp}
+            onDelete={() => {
+              const savedKeypairs = localStorageSavedKeypairs.get();
+              const indexToUpdate = savedKeypairs.findIndex(
+                (kp) => kp.timestamp === keypair.timestamp,
+              );
+
+              if (indexToUpdate >= 0) {
+                const updatedList = arrayItem.delete(
+                  savedKeypairs,
+                  indexToUpdate,
+                );
+
+                localStorageSavedKeypairs.set(updatedList);
+                updateSavedKeypairs();
+              }
+            }}
+          />
+        </Box>
+      </Box>
+    );
+  };
+
+  const getNetworkById = (networkId: NetworkType) => {
+    const newNetwork = NetworkOptions.find((n) => n.id === networkId);
+
+    if (newNetwork) {
+      updateIsDynamicNetworkSelect(true);
+      selectNetwork(newNetwork);
+    }
+  };
+
+  const renderOtherNetworkMessage = () => {
+    if (network.id === "testnet" || network.id === "futurenet") {
+      const otherNetworkLabel =
+        network.id === "testnet" ? "Futurenet" : "Testnet";
+
+      return (
+        <Alert
+          variant="primary"
+          title={`Looking for ${otherNetworkLabel} keypairs?`}
+          placement="inline"
+          actionLabel={`Switch to ${otherNetworkLabel}`}
+          onAction={() => {
+            const newNetworkId =
+              network.id === "testnet" ? "futurenet" : "testnet";
+
+            getNetworkById(newNetworkId);
+          }}
+        >
+          {`You must switch your network to ${otherNetworkLabel} in order to see those saved
```

Comment:
can we just do the following? 
`You must switch your network to {otherNetworkLabel} in order to see those saved keypairs. This feature is only available on Futurenet and Testnet for security reasons.`

Reply:
Yes, that works, too. In cases like this, I prefer formatting it as a string because it's more clear and consistent in the code editor what it is. This is just my preference, nothing else. :blush: If you feel strongly about it, I can update it.

![image](https://github.com/user-attachments/assets/6ffcd031-a23c-42d5-b0fa-b8cfd874e1fd)

vs

![image](https://github.com/user-attachments/assets/627a3b7c-84b6-4564-9d07-020085f10971)